Documentation/policy/language.rst

```diff
@@ -747,6 +747,9 @@ latter rule will have no effect.
           endpoint. This might change in the future when support for ranges is
           added.
```

Comment:
(Not related to this PR)

@jrajahalme do we still have this limit of 40 ports with L7 policies per endpoint? Seems like the kind of limitation that tproxy got rid of.